Server/src/quadradiusr_server/rest/lobby_views.py

```diff
@@ -8,26 +8,36 @@
 from quadradiusr_server.db.base import Lobby, User, LobbyMessage
 from quadradiusr_server.db.repository import Repository
 from quadradiusr_server.db.transactions import transactional, transaction_context
-from quadradiusr_server.lobby import LobbyConnection
+from quadradiusr_server.lobby import LobbyConnection, LiveLobby
 from quadradiusr_server.notification import NotificationService
 from quadradiusr_server.qrws_connection import QrwsConnection
 from quadradiusr_server.rest.auth import authorized_endpoint
 from quadradiusr_server.server import routes, QuadradiusRServer
 
 
-def map_lobby_to_json(server: QuadradiusRServer, lobby: Lobby):
+def map_lobby_to_json(
+        server: QuadradiusRServer,
+        lobby: Lobby,
+        live_lobby: LiveLobby = None):
     return {
         'id': lobby.id_,
         'name': lobby.name_,
         'ws_url': server.get_href('ws') + f'/lobby/{lobby.id_}/connect',
+        'players': [{
```

Comment:
I wonder if it wouldn't be beneficial for us to add nicknames as well, since I think in all usecases when requesting this list we want to show it to user somewhere